src/pages/team-invite/[token].tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from 'react'
```

Comment:
This page is what you see in the above description: this is where the user goes when they visit a generic team invite URL.